README.md

```diff
@@ -324,7 +324,7 @@ This software has been approved for open source release and has been assigned **
 © (or copyright) 2020. Triad National Security, LLC. All rights reserved.  This
 program was produced under U.S. Government contract 89233218CNA000001 for Los
 Alamos National Laboratory (LANL), which is operated by Triad National
-Security, LLC for the U.S.  Department of Energy/National Nuclear Security
+Security, LLC for the U.S. Department of Energy/National Nuclear Security
```

Comment:
I was keeping it exactly as they gave it to me ;) (jk I'm sure it does not matter at all)